python/lsst/pex/config/comparison.py

```diff
@@ -59,7 +59,15 @@ def getComparisonName(name1, name2):
     return name1
 
 
-def compareScalars(name, v1, v2, output, rtol=1e-8, atol=1e-8, dtype=None):
+def compareScalars(
```

Comment:
What caused this to be reformatted?

Reply:
I'd added some extra arguments that I later regretted and removed.  Restored.